pubspec.yaml

```diff
@@ -2,14 +2,27 @@ name: dart_to_js_script_rewriter
 description: Replaces Dart script tags with JavaScript script tags,
              speeding up initial loads and reducing 404s. Use when
              ready to deploy.
-author: Seth Ladd <sethladd@gmail.com>
```

Comment:
Do we have a pattern for that, or should I just add a line to the very end of the readme

`Originally authored by Seth Ladd <sethladd@gmail.com>`